autoPlay.js

```diff
@@ -55,24 +55,20 @@ var ENEMY_TYPE = {
 
 function firstRun() {
 	// disable particle effects - this drastically reduces the game's memory leak
-	if (window.g_Minigame !== undefined) {
-		window.g_Minigame.CurrentScene().DoClickEffect = function() {};
-		window.g_Minigame.CurrentScene().DoCritEffect = function( nDamage, x, y, additionalText ) {};
-		window.g_Minigame.CurrentScene().SpawnEmitter = function(emitter) {
+	if (g_Minigame !== undefined) {
+		g_Minigame.CurrentScene().DoClickEffect = function() {};
+		g_Minigame.CurrentScene().DoCritEffect = function( nDamage, x, y, additionalText ) {};
+		g_Minigame.CurrentScene().SpawnEmitter = function(emitter) {
 			emitter.emit = false;
 			return emitter;
 		}
 	}
 
 	// disable enemy flinching animation when they get hit
-	if (window.CEnemy !== undefined) {
-		window.CEnemy.prototype.TakeDamage = function() {};
-		window.CEnemySpawner.prototype.TakeDamage = function() {};
-		window.CEnemyBoss.prototype.TakeDamage = function() {};
-	}
-
-	if (thingTimer !== undefined) {
```

Comment:
Put the `clearTimeout(thingTimer)` at the beginning of the script; it's there to clear the old version before starting the latest version.

If it needs to not be run for GM/TM, add a condition so it only runs for manual.